modules/ui/fields/localized.js

```diff
@@ -116,6 +116,9 @@ export function uiFieldLocalized(field, context) {
 
     // update _multilingual, maintaining the existing order
     function calcMultilingual(tags) {
+        // tags that use the format name:<item>, but
+        // aren't languages
+        const nonLanguageTags = ["left", "right", "source", "signed", "etymology"]
```

Comment:
This is a good start, though quite a few common non-language subkeys could be added to this list, either in this PR or in a followup. Off the top of my head, there’s also [`pronunciation`](https://wiki.openstreetmap.org/wiki/Key:name:pronunciation)[^fonipa], [`prefix`](https://wiki.openstreetmap.org/wiki/Key:name:prefix), and `genitive`.

To avoid a slow trickle of additions, perhaps we could match only the subkeys that follow the BCP&nbsp;47 pattern. The following regular expression is similar to the one StreetComplete introduced in streetcomplete/StreetComplete@04fd11acebcb17ef050696164ec583fecd843545, but it also covers country-specific subkeys, such as the Hong Kong and Taiwan variants of Traditional Chinese.

```regex
[a-z]{2,3}([A-Z][a-z]{3})?([A-Z]{2})?
```

Another syntax for transliteration is still in widespread use, for example `name:ja_rm` for romaji, but it’s deprecated, so we don’t need to go out of our way to support it.

/ref https://github.com/streetcomplete/StreetComplete/issues/2723#issuecomment-815057365

[^fonipa]: Technically, `name:pronunciation` should be `name:*-fonipa`, but no one is using that syntax.